content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -368,41 +368,22 @@ Here's a breakdown of what/why:
    certificates from the `kube-apiserver` when the certificate expiration approaches.
 * `--cert-dir`the directory where the TLS certs are located.
 
-### Use kubeadm with other CRI runtimes
+### Use kubeadm with containerd
```

Comment:
I'd propose to leave this as it is and add setup instructions for at least containerd and CRI-O.

Reply:
looked at [rktlet](https://github.com/kubernetes-incubator/rktlet) and [frakti](https://github.com/kubernetes/frakti). Not sure I understand why to remove them from this document.

Reply:
We could add instructions for other runtimes with kubeadm to this page later on, does that sound good? We should definitely track these.